### PR TITLE
Debug mode

### LINKED ISSUE #88: [FEATURE] Debug mode
In debug mode:

- A new random game will appear every time the page refreshes
- User will have the option to delete their localStorage stats.

### PR DESCRIPTION
**Pull request by lpatmo**
## Description
This PR resolves #88 

image

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -24,17 +26,39 @@ import { differenceInDays } from 'date-fns';
...
-    let randomIndex = Math.floor(Math.random() * data.length)
-    setAnswer(data[setGameIndex()]);
+    
+    const urlSearchParams = new URLSearchParams(window.location.search);
+    const searchParamsObject = {};
+    // Iterate over each search parameter and store the values in the object
+    for (const [key, value] of urlSearchParams) {
+      searchParamsObject[key] = value;
+    }
+    if (searchParamsObject.testMode === 'true') {
```
Comment (arturolei) [inline src/App.jsx:54]: Might want to add comment about how to enter testmode, e.g. append `?testMode=true` to URL

**Review by arturolei — APPROVED** [review]
Looks good. See comment. We might want to revisit this later with routing.

**Review by ISPOL — APPROVED** [review]
The URL param is currently case sensitive, not sure if that matters or if we should change that.

Otherwise looks fine.

**Comment by lpatmo** [comment]
@ISPOL Thanks! I think it's fine to be case-sensitive because we want debug mode to be somewhat hidden.

**Pull request merged**